README.md

```diff
@@ -128,12 +128,31 @@ go install github.com/charmbracelet/vhs@latest
 
 ## The VHS Server
 
-VHS has an SSH server built in! When you self host VHS you can access it as
+VHS has an SSH server built in! When you self-host VHS you can access it as
 though it were installed locally. VHS will have access to commands and
 applications on the host so you don't need to install them on your machine.
 
-Ensure you've set `~/.ssh/authorized_keys` with the public keys to which
-you want grant SSH access. You can change the file VHS looks at by setting
+<details>
+<summary>
+
+### Server Configuration
```

Comment:
I think the title looks a bit weird inside a summary... would probably change it to 

```html
<summary>Server Configuration</summary>
``` 

only